File: source/frontend/src/app/entities/vacature/antwoordnummerAdres.ts
import {AntwoordnummerAdresImpl} from "./antwoordnummerAdresImpl";

export interface AntwoordnummerAdres {
  antwoordnummeradres?: AntwoordnummerAdresImpl;
}

export function isAntwoordnummeradres(object: any): object is AntwoordnummerAdres {
  return 'antwoordnummeradres' in object;
}

File: source/backend-vacatures-bemiddelaar/src/main/java/com/cap/pocvng/entity/Antwoordnummeradres.java
package com.cap.pocvng.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.validation.Valid;

/**
 * Antwoordnummeradres
 */


@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Antwoordnummeradres implements AdresDetailsNederland {

    @Valid
    private AntwoordnummeradresImpl antwoordnummeradres;

}
